bdai_ros2_wrappers/bdai_ros2_wrappers/action_handle.py

```diff
@@ -123,21 +123,29 @@ def get_feedback_callback(self, feedback: Action.Feedback) -> None:
     def _get_result_callback(self, future: Future) -> None:
         """Callback for pulling the resulting feedback out of the future and passing it to a user provided callback"""
         result = future.result()
+        if result is not None:
```

Comment:
+1 on this. `result is None` seems to indicate the check of `if result.status == ...` will crash. Should we be returning or something instead of continuing with the checks?